build.sbt

```diff
@@ -153,6 +159,7 @@ lazy val ratatoolCommon = project
       "org.apache.avro" % "avro" % avroVersion,
       "org.apache.avro" % "avro-mapred" % avroVersion classifier "hadoop2",
       "com.google.guava" % "guava" % guavaVersion,
+      "org.apache.beam" % "beam-vendor-guava-32_1_2-jre" % beamVendorVersion,
```

Comment:
I don't think common should depend on beam. It is fine to use non shaded guava IMHO